### PR TITLE
Minor language updates.

### PR DESCRIPTION
**Pull request by gaurav-nelson**
### What does this PR do?
Some minor language update suggestions.
- Removes some passive voice usage.
- Removes some fluff which is not needed.

### Motivation
Improving existing documentation.

### CODE REVIEW

File: content/agent/_index.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ Agent v6 is available. <a href="https://docs.datadoghq.com/agent/faq/upgrade-to-
 
 ## What is the Agent?
 
-The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open source: view the source code on GitHub for [Agent v6][2].
+The Datadog Agent is a software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog. From where you can analyze your monitoring and performance data. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][2].
```
Comment (gaurav-nelson) [inline content/agent/_index.md]: From history it was visible that the link to the GitHub repo was recently changed to version 6. Having a direct link to the repo without mentioning version will save us from remembering to update it with every new version.

File: content/agent/_index.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ Agent v6 is available. <a href="https://docs.datadoghq.com/agent/faq/upgrade-to-
...
 
 {{< partial name="platforms/platforms.html" >}}
 
-The Agent has three main parts: the collector, DogStatsD, and the forwarder:
+The Agent has three main parts: the **collector**, **DogStatsD**, and the **forwarder**:
 
-* **Collector**: Runs checks on the current machine for whatever [integrations][4] you have, and captures system metrics such as memory and CPU.
-* **DogStatsD**: A StatsD-compatible backend server that you can send [custom metrics][5] to from your own applications.
-* **Forwarder**: Retrieves data from both DogStatsD and the collector, and then queues it up to be sent to Datadog.
+1. **Collector**: It runs checks on the current machine for configured [integrations][4], and captures system metrics, such as memory and CPU.
```
Comment (gaurav-nelson) [inline content/agent/_index.md]: Changed it to numbered list because we are calling it three main parts.

File: content/agent/_index.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ The new command line interface for the Agent v6 is sub-command based:
 | `version`         | Print the version info                                                     |
 
 
-To run a sub-command, the Agent binary must be invoked like this:
+To run a sub-command, first invoke the Agent binary.
```
Comment (gaurav-nelson) [inline content/agent/_index.md]: I prefer using `call` instead of invoke, but kept it still.

File: content/agent/_index.md [hunk 4]
```diff
@@ -71,19 +71,20 @@ Some options have their own set of flags and options detailed in a help message.
 
 ## Using the GUI
```
Comment (gaurav-nelson) [inline content/agent/_index.md]: The section title is `Using the GUI` but its missing the usage part. We should either:
- change the title to just `GUI` (similar to the CLI section)
- Or add information on how to use the GUI (or refer to an existing section which does that)

Comment (cswatt) [reply content/agent/_index.md]: Let's go with `GUI`

**Comment by gaurav-nelson** [comment]
Thank you @cswatt for reviewing this. I have updated my commit to include suggested changes.

**Pull request merged**